CWE/events/italy.txt

```diff
@@ -455,6 +455,1043 @@ country_event = {
+	}
+
+	mean_time_to_happen = { months = 1 }
+
+	option = {
+		name = "Maintain the hardline"
+		
+		ruling_party_ideology = conservative
+		
+		any_pop = { 
+			limit = { is_primary_culture = yes OR = { strata = rich strata = middle } }
+			ideology = { factor = 0.01 value = nationalist } 
+		}
+		any_pop = {
+			scaled_consciousness = { factor = 4 ideology = communist }
+			scaled_consciousness = { factor = 4 ideology = communist_social }
+			scaled_consciousness = { factor = 4 ideology = socialist }
+			scaled_militancy = { factor = 6 ideology = communist }
+			scaled_militancy = { factor = 6 ideology = communist_social }
+			scaled_militancy = { factor = 6 ideology = socialist }
+		}
+		
+		remove_country_modifier = political_strife
+		
+		add_country_modifier = { name = political_strife duration = 365 }
+		set_country_flag = ITA_1960_Genoa_Hardline
+		
+		ai_chance = { factor = 0.1 }
+	}
+	
+	option = {
+		name = "Resign"
+		
+		ruling_party_ideology = big_tent
+		
+		#Leadership
+		remove_country_modifier = expert_diplomat
+		remove_country_modifier = successful_industrialist
+		remove_country_modifier = ivory_tower_intellectual
+		remove_country_modifier = man_of_the_people
+		remove_country_modifier = landed_gentleman
+		remove_country_modifier = military_man
+		remove_country_modifier = lord_admiral
+		remove_country_modifier = raving_loon
+		remove_country_modifier = luddite
+		remove_country_modifier = great_statesman
+		remove_country_modifier = backroom_boy
+		remove_country_modifier = great_reformer
+		remove_country_modifier = gauche_buffoon
+		remove_country_modifier = administrative_genius
+		remove_country_modifier = visionary_thinker
+		remove_country_modifier = competent_placeholder
+		remove_country_modifier = imperious_autocrat
+		remove_country_modifier = prince_of_terror
+		remove_country_modifier = efficient_sociopath
+		remove_country_modifier = pig_headed_isolationist
+		remove_country_modifier = ambitious_union_boss
+		remove_country_modifier = dour_economist
+		
+		add_country_modifier = great_reformer 
+		
+		any_pop = { 
+			ideology = { factor = -0.05 value = conservative } 
+			ideology = { factor = 0.07 value = big_tent } 
+			ideology = { factor = 0.01 value = liberal } 
+		}
+		any_pop = { 
+			limit = { strata = middle }
+			ideology = { factor = 0.03 value = socialist }
+		}
+		any_pop = {
+			scaled_consciousness = { factor = 2 ideology = communist }
+			scaled_consciousness = { factor = 2 ideology = communist_social }
+			scaled_consciousness = { factor = 2 ideology = socialist }
+			scaled_militancy = { factor = 4 ideology = communist }
+			scaled_militancy = { factor = 4 ideology = communist_social }
+			scaled_militancy = { factor = 2 ideology = socialist }
+		}
+		
+		remove_country_modifier = political_strife
+		add_country_modifier = { name = political_strife duration = 365 }
+		
+		set_country_flag = ITA_Democratic_Renovation
+		
+		ai_chance = { factor = 0.9 }
```

Comment:
Event option with highest chance should be ordered first in the event's list of options.